app/models/account.rb

```diff
@@ -134,11 +134,11 @@ def subscribed?
   end
 
   def keypair
-    OpenSSL::PKey::RSA.new(private_key || public_key)
+    @keypair ||= OpenSSL::PKey::RSA.new(private_key || public_key)
   end
 
   def subscription(webhook_url)
-    OStatus2::Subscription.new(remote_url, secret: secret, lease_seconds: 30.days.seconds, webhook: webhook_url, hub: hub_url)
+    @subscription ||= OStatus2::Subscription.new(remote_url, secret: secret, webhook: webhook_url, hub: hub_url)
```

Comment:
I removed it because it would confuse readers. The subscribe/unsubscribe methods of `OStatus2::Subscription` are no longer used anywhere. SubscribeService defines its own lease_seconds.